src/main/java/me/botsko/prism/commands/TeleportCommand.java

```diff
@@ -135,13 +132,25 @@ public void handle(CallInfo call) {
                         Prism.messenger.playerError("Action record occurred in world we can't find anymore."));
                 return;
             }
+            if (PaperLib.isPaper()) {
+                PaperLib.teleportAsync(call.getPlayer(), destinationAction.getLoc())
+                        .thenAccept(
+                              success -> sendTeleportCompleteMessage(success, call.getPlayer(), destinationAction));
+            } else {
+                sendTeleportCompleteMessage(call.getPlayer().teleport(destinationAction.getLoc()), call.getPlayer(),

```

Comment:
Personally, I would split this into two statemets
```java
boolean success = call.getPlayer().teleport(destinationAction.getLoc());
sendTeleportCompleteMessage(success, call.getPlayer(), destinationAction);
```

Reply:
you cant
....because its aysnc in one case

Reply:
oh right you mean the last statement